sci-physics/rivet/rivet-2.5.2.ebuild

```diff
@@ -0,0 +1,75 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit python-single-r1 autotools bash-completion-r1
+
+MYP=Rivet-${PV}
```

Comment:
instead of using `"Rivet-${PV}"`, use `${P^}` in `EAPI=6` (capitalise the first letter of `$P`). Hence replace all these MYP cases